### ISSUE TITLE
Please give us simple example in plain English

### ISSUE THREAD

**Issue by yoskakomba** [FreeCodeCamp/FreeCodeCamp#3588]
Hi, thank you for the tutorial.
If you don't mind, would you mind give us simple examples in plain English with detailed definition as many campers are non English native speaker. 
thank you

**Comment by ahstro** [comment]
I don't think the English version of the site is going to be simplified on a large scale; if you find anything especially confusing, open an issue for it.
However, translations of freeCodeCamp are a thing, and so are language-specific chat rooms.
I'm not sure how translations work, but here are the related wiki pages I could find:
[Translating Free Code Camp into Your Native Language](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Translating-Free-Code-Camp-into-Your-Native-Language)
[FreeCodeCamp completion language requirements](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Free-Code-Camp-completion-language-requirements)
[What to do if you speak a language that Free Code Camp does not yet support](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/What-to-do-if-you-speak-a-language-that-Free-Code-Camp-does-not-yet-support)
[World Language Chat Rooms](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/World-Language-Chat-Rooms)
